resources/nl-core/nl-core-BodyWeight.xml

```diff
@@ -0,0 +1,31 @@
+<?xml version="1.0" encoding="utf-8"?>
+<StructureDefinition xmlns="http://hl7.org/fhir">
+  <id value="nl-core-BodyWeight" />
+  <url value="http://nictiz.nl/fhir/StructureDefinition/nl-core-BodyWeight" />
+  <name value="NlcoreBodyWeight" />
+  <title value="nl core BodyWeight" />
+  <status value="draft" />
+  <publisher value="Nictiz" />
+  <contact>
+    <name value="Nictiz" />
+    <telecom>
+      <system value="email" />
+      <value value="info@nictiz.nl" />
+      <use value="work" />
+    </telecom>
+  </contact>
+  <description value="Body weight is the common name for (human) body mass." />
+  <purpose value="A derived profile from [zib-BodyWeight](http://nictiz.nl/fhir/StructureDefinition/zib-BodyWeight) to provide a version better suited for implementation purposes. This profile augments the base profile with elements found in the various use cases that have adopted the zib." />
+  <copyright value="Copyright and related rights waived via CC0, https://creativecommons.org/publicdomain/zero/1.0/. This does not apply to information from third parties, for example a medical terminology system. The implementer alone is responsible for identifying and obtaining any necessary licenses or authorizations to utilize third party IP in connection with the specification or otherwise." />
+  <fhirVersion value="4.0.1" />
+  <mapping>
+    <identity value="zib-bodyweight-v3.2-2020EN" />
+    <uri value="https://zibs.nl/wiki/BodyWeight-v3.2(2020EN)" />
+    <name value="zib BodyWeight-v3.2(2020EN)" />
+  </mapping>
+  <kind value="resource" />
+  <abstract value="false" />
+  <type value="Observation" />
+  <baseDefinition value="http://nictiz.nl/fhir/StructureDefinition/zib-BodyWeight" />
+  <derivation value="constraint" />
```

Comment:
Alias on the root is missing

Reply:
Added nl-core-AnatomicalLocation on the root